### PR TITLE
Add env specific config for housing repairs online api

### PR DESCRIPTION
**Pull request by maysakanoni**

### CODE REVIEW

**Comment by maysakanoni** [comment]
[Allow differentiated configuration between staging and production for HousingRepairsOnlineApi](https://trello.com/c/OcUwkiwk/125-allow-differentiated-configuration-between-staging-and-production-for-housingrepairsonlineapi)

File: docs/adoption.md [hunk 1]
```diff
@@ -167,29 +167,33 @@ Finally, in GitHub actions secrets, set `AZUREAPPSERVICE_PUBLISHPROFILE_PRODUCTI
...
-| `GOV_NOTIFY_KEY_STAGING`                | Production gov notify key                                                                                    |
-| `INTERNAL_EMAIL`                        | Internal email address for receiving repair request details, for any manual follow-on process                |
+| `DAYS_UNTIL_IMAGE_EXPIRY_PRODUCTION`    | Number in days before image uploaded by customer expires for _Production_, e.g. `14` days                    |
+| `DAYS_UNTIL_IMAGE_EXPIRY_STAGING`       | Number in days before image uploaded by customer expires for _Staging_, e.g. `14` days                       |
+| `GOV_NOTIFY_KEY_PRODUCTION`             | _Staging_ gov notify key                                                                                     |
+| `GOV_NOTIFY_KEY_STAGING`                | _Production_ gov notify key                                                                                  |
+| `INTERNAL_EMAIL_PRODUCTION`             | Internal email address for receiving repair request details, for any manual follow-on process in production  |
```
Comment (Bhavesh-Patel) [inline docs/adoption.md:183]: Update the environment text style for consistency with other changes.
```suggestion
| `INTERNAL_EMAIL_PRODUCTION`             | Internal email address for receiving repair request details, for any manual follow-on process in _Production_  |
```

File: docs/adoption.md [hunk 2]
```diff
@@ -167,29 +167,33 @@ Finally, in GitHub actions secrets, set `AZUREAPPSERVICE_PUBLISHPROFILE_PRODUCTI
...
+| `INTERNAL_EMAIL_STAGING`                | Internal email address for receiving repair request details, for any manual follow-on process in staging     |
```
Comment (Bhavesh-Patel) [inline docs/adoption.md:184]: Update the environment text style for consistency with other changes.
```suggestion
| `INTERNAL_EMAIL_STAGING`                | Internal email address for receiving repair request details, for any manual follow-on process in _Staging_     |
```

**Review by Bhavesh-Patel — CHANGES_REQUESTED** [review]
Couple of minor change requests

**Review by Bhavesh-Patel — APPROVED** [review]

**Review by jamiejordanmt — APPROVED** [review]

**Pull request merged**